Connecting to TiDB with VS Code: added instructions on how to install SQLTools MySQL/MariaDB/… (#16133)
pingcap/docs: develop/dev-guide-gui-vscode-sqltools.md

- [SQLTools MySQL/MariaDB/TiDB](https://marketplace.visualstudio.com/items?itemName=mtxr.sqltools-driver-mysql) extension for Visual Studio Code. To install it, you can use one of the following methods:
    - Click <a href="vscode:extension/mtxr.sqltools-driver-mysql">this link</a>  to launch VS Code and install the extension directly.
    - Navigate to [VS Code Marketplace](https://marketplace.visualstudio.com/items?itemName=mtxr.sqltools-driver-mysql) and click **Install**.

>     - In the extensions tab of your VS Code, search for **SQLTools MySQL/MariaDB/TiDB**. Select the matching item from the displayed search result and click **Install**.

Reviewer: Wouldn't it be better to search for the name of the extension in case some other extension gets published that also matches this search?

![image](https://github.com/pingcap/docs/assets/1272980/1afefc20-5d05-4083-9af9-3d04c78b2cdb)

Reviewer: Another option, Ctrl-+P and then enter this command:
```
ext install mtxr.sqltools-driver-mysql
```

![image](https://github.com/pingcap/docs/assets/1272980/2348b42f-7e3c-42ba-87fa-c5ede19cb2dd)

Reviewer 2: Agree, searching with `mtxr.sqltools-driver-mysql` is more efficient. As for "Ctrl-+P", as "Ctrl" varies slight among different operation systems, I think it is OK not to mention that.  Hence, here are my suggested changes. @kenudeh, PTAL

```suggestion
    - In the extensions tab of your VS Code, search `mtxr.sqltools-driver-mysql` to get the **SQLTools MySQL/MariaDB/TiDB**  exertion and click **Install**. 
```

Author: @dveeden, yes, it's a lot better to search using the extension's name. In addition to the remark @qiancai made about your second suggestion, I think using the search feature to install extensions in VS Code is a more common practice.
 @qiancai, thank you for the suggested changes. I sent this PR in haste to beat the deadline during the Docs Dash. I'll commit them now.
